### PR TITLE
add SLE15 SP3 image descriptions

### PR DESCRIPTION
**Pull request by jgleissner**
Also includes GCE fence agent dependencies.

### CODE REVIEW

File: images/sles-sap/azure-li/15-sp3/image.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp1
+  - sle15/sp2
+  - sle15/sp3
+image:
+  name: SLES15-SP3-SAP-Azure-LI-BYOS
+  specification: "SUSE Linux Enterprise Server for SAP Applications 15 SP3 Azure LI BYOS guest image"
+  version: 1.0.42
```
Comment (rjschwei) [inline images/sles-sap/azure-li/15-sp3/image.yaml:8]: While not in production version number should be 0.9.x see the "Image Versioning" document in our confluence area.

File: images/sles-sap/azure-vli/15-sp3/image.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp1
+  - sle15/sp2
+  - sle15/sp2
+image:
+  name: SLES15-SP3-SAP-Azure-VLI-BYOS
+  specification: "SUSE Linux Enterprise Server for SAP Applications 15 SP3 Azure VLI BYOS guest image"
+  version: 1.0.42
```
Comment (rjschwei) [inline images/sles-sap/azure-vli/15-sp3/image.yaml:8]: Ditto

File: images/sles-sap/byos/15-sp3/image.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp1
+  - sle15/sp2
+  - sle15/sp3
+image:
+  name: SLES15-SP3-SAP-BYOS
+  specification: "SUSE Linux Enterprise Server for SAP Applications 15 SP3 BYOS guest image"
+  version: 1.0.42
```
Comment (rjschwei) [inline images/sles-sap/byos/15-sp3/image.yaml:8]: Ditto

File: images/sles-sap/ondemand/15-sp3/image.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp1
+  - sle15/sp2
+  - sle15/sp3
+image:
+  name: SLES15-SP3-SAP
+  specification: "SUSE Linux Enterprise Server for SAP Applications 15 SP3 guest image"
+  version: 1.0.42
```
Comment (rjschwei) [inline images/sles-sap/ondemand/15-sp3/image.yaml:8]: Ditto

File: images/sles/byos/15-sp3/image.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp1
+  - sle15/sp2
+  - sle15/sp3
+image:
+  name: SLES15-SP3-BYOS
+  specification: "SUSE Linux Enterprise Server 15 SP3 BYOS guest image"
+  version: 1.0.42
```
Comment (rjschwei) [inline images/sles/byos/15-sp3/image.yaml:8]: DItto

File: images/sles/chost/15-sp3/image.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp1
+  - sle15/sp2
+  - sle15/sp3
+image:
+  name: SLES15-SP3-CHOST-BYOS
+  specification: "SUSE Linux Enterprise Server 15 SP3 BYOS guest image optimized as container host"
+  version: 1.0.42
```
Comment (rjschwei) [inline images/sles/chost/15-sp3/image.yaml:8]: Ditto

File: images/sles/ec2-ecs/15-sp3/image.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp1
+  - sle15/sp2
+  - sle15/sp3
+image:
+  name: SLES15-SP3-EC2-ECS-HVM
+  specification: "SUSE Linux Enterprise Server 15 SP3 ECS guest image for Amazon EC2 with HVM"
+  version: 1.0.42
```
Comment (rjschwei) [inline images/sles/ec2-ecs/15-sp3/image.yaml:8]: Ditto

File: images/sles/hpc-byos/15-sp3/image.yaml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp1
+  - sle15/sp2
+  - sle15/sp3
+image:
+  name: SLES15-SP3-HPC
+  specification: "SUSE Linux Enterprise Server 15 SP3 HPC guest image"
+  version: 1.0.42
```
Comment (rjschwei) [inline images/sles/hpc-byos/15-sp3/image.yaml:8]: DItto

File: images/sles/hpc/15-sp3/image.yaml [hunk 9]
```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp1
+  - sle15/sp2
+  - sle15/sp3
+image:
+  name: SLES15-SP3-HPC
+  specification: "SUSE Linux Enterprise Server 15 SP3 HPC guest image"
+  version: 1.0.42
```
Comment (rjschwei) [inline images/sles/hpc/15-sp3/image.yaml:8]: Ditto

File: images/sles/ondemand/15-sp3/image.yaml [hunk 10]
```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp1
+  - sle15/sp2
+  - sle15/sp3
+image:
+  name: SLES15-SP3
+  specification: "SUSE Linux Enterprise Server 15 SP3 guest image"
+  version: 1.0.42
```
Comment (rjschwei) [inline images/sles/ondemand/15-sp3/image.yaml:8]: DItto

**Review by rjschwei — APPROVED** [review]
Just the version settings, needs no additional review after fix.

**Comment by schaefi** [comment]
@rjschwei I've updated the version setting as you suggested

**Pull request merged**